bin/agent-data-plane/src/main.rs

```diff
@@ -200,10 +206,15 @@ fn create_topology(
         .add_transform("enrich", enrich_config)?
         .add_destination("dd_metrics_out", dd_metrics_config)?
         .add_destination("dd_events_sc_out", events_service_checks_config)?
+        .add_destination("dd_status_flare_out", _status_configuration)?
         .connect_component("dsd_agg", ["dsd_in.metrics"])?
         .connect_component("enrich", ["dsd_agg"])?
         .connect_component("dd_metrics_out", ["enrich"])?
-        .connect_component("dd_events_sc_out", ["dsd_in.events", "dsd_in.service_checks"])?;
+        .connect_component("dd_events_sc_out", ["dsd_in.events", "dsd_in.service_checks"])?
+        .connect_component(
+            "dd_status_flare_out",
+            ["enrich", "dsd_in.events", "dsd_in.service_checks"],
```

Comment:
What we really want to do is actually feed this with internal metrics.

We should probably copy all the non-Prometheus bits out of the if block below, related to internal telemetry, add them here, and then change the if block to just create the Prometheus destination and wire it up to `internal_metrics_remap`.